### PR TITLE
[FIX] Text area buttons and layout on mobile 

### LINKED ISSUE #7935: message box on mobile/small screens
image

image

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core 

Closes #7935

### CODE REVIEW

File: packages/rocketchat-theme/client/imports/components/message-box.css [hunk 1]
```diff
@@ -174,8 +181,58 @@
 		&__container {
 			display: flex;
 
+			padding: 1rem 1rem calc(1rem - 8px) 1rem;
```
Comment (sampaiodiego) [inline packages/rocketchat-theme/client/imports/components/message-box.css]: do we need this `calc`?

File: packages/rocketchat-ui-message/client/messageBox.js [hunk 2]
```diff
@@ -153,7 +158,12 @@ Template.messageBox.helpers({
 		});
 
 		return columns;
-	},
+	}
+};
+
+Template.messageBox__actions.helpers({...methods});
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-message/client/messageBox.js]: you can pass `methods` variable directly, don't need to spread it into another object

File: packages/rocketchat-ui-message/client/messageBox.js [hunk 3]
```diff
@@ -153,7 +158,12 @@ Template.messageBox.helpers({
...
+Template.messageBox__actionsSmall.helpers({...methods});
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-message/client/messageBox.js]: same as above

File: packages/rocketchat-ui-message/client/messageBox.js [hunk 4]
```diff
@@ -275,8 +285,9 @@ Template.messageBox.helpers({
 	anonymousWrite() {
 		return (Meteor.userId() == null) && RocketChat.settings.get('Accounts_AllowAnonymousRead') === true && RocketChat.settings.get('Accounts_AllowAnonymousWrite') === true;
 	},
-	sendIcon() {
-		return Template.instance().sendIcon.get();
+	disableSendIcon() {
+		console.log(Template.instance().sendIcon.get());
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-message/client/messageBox.js]: :man_shrugging:

File: packages/rocketchat-ui/client/lib/textarea-autogrow.js [hunk 5]
```diff
@@ -9,8 +9,7 @@
 		return this.filter('textarea').each(function() {
 			var self = this;
 			var $self = $(self);
-			var minHeight = $self.height();
-			var noFlickerPad = $self.hasClass('autogrow-short') ? 0 : parseInt($self.css('lineHeight')) || 0;
+			var minHeight = 16; //$self.height();
```
Comment (sampaiodiego) [inline packages/rocketchat-ui/client/lib/textarea-autogrow.js]: is this intentional?

File: packages/rocketchat-ui/client/lib/textarea-autogrow.js [hunk 6]
```diff
@@ -54,9 +56,9 @@
 				}
 
 				shadow.css('width', $self.width());
-				shadow.html(val + (noFlickerPad === 0 ? '...' : '')); // Append '...' to resize pre-emptively.
+				shadow.html(val); // Append '...' to resize pre-emptively.
```
Comment (sampaiodiego) [inline packages/rocketchat-ui/client/lib/textarea-autogrow.js]: the comment does not make sense anymore

File: packages/rocketchat-ui-message/client/messageBox.js [hunk 7]
```diff
@@ -130,7 +130,12 @@ const markdownButtons = [
 	}
 ];
 
-Template.messageBox.helpers({
+const methods = {
+	actions() {
+		const groups = RocketChat.messageBox.actions.get();
+		console.log(Object.keys(groups).reduce((ret, el) => ret.concat(groups[el]), []));
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-message/client/messageBox.js]: :man_shrugging:

File: packages/rocketchat-ui/client/lib/textarea-autogrow.js (hunk 6 above)
Comment (ggazzo) [reply packages/rocketchat-ui/client/lib/textarea-autogrow.js]: your brain gets smart but your head gets dumb 🎶

File: packages/rocketchat-ui-message/client/messageBox.js (hunk 2 above)
Comment (ggazzo) [reply packages/rocketchat-ui-message/client/messageBox.js]: I know, but we are suggesting that we can put more functions inside.

File: packages/rocketchat-ui-message/client/messageBox.js (hunk 3 above)
Comment (ggazzo) [reply packages/rocketchat-ui-message/client/messageBox.js]: 💩

File: packages/rocketchat-ui-message/client/messageBox.js (hunk 2 above)
Comment (ggazzo) [reply packages/rocketchat-ui-message/client/messageBox.js]: I`m sorry sir @sampaio.diego, you are right.

**Pull request merged**